intelmq/lib/harmonization.py

```diff
@@ -700,6 +700,21 @@ def to_ip(value: str) -> Optional[str]:
             value = None
         return value
 
+    @staticmethod
+    def to_domain_suffix(value: str) -> Optional[str]:
+
+        value = FQDN.sanitize(value)
+
+        if value:
+
+            parts = value.split(".")
+
+            if len(parts) > 1:
+                return parts[-1]
```

Comment:
The "domain suffix" is not just the last part of the domain. See also https://intelmq.readthedocs.io/en/latest/user/bots.html#domain-suffix for an existing bot which already covers this functionality.

Reply:
Removed. Thank you, I missed this bot.